src/lib/watchdog.coffee

```diff
@@ -40,7 +40,11 @@ class Watchdog extends EventEmitter
   # Fetch tasks
   run: () ->
     async.forever (done) =>
-      return done new QueuerErrors.ShutdownError() if @isShuttingDown
+      if @isShuttingDown
+        @notifyHealthStatusChange 'unhealthy'
```

Comment:
`'healthy'` and `'unhealthy'` strings are used twice in this PR, perhaps it would be good to extract constants?

Reply:
👍